pyproject.toml

```diff
@@ -25,9 +27,14 @@ prettytable = "^3.3.0"
 vt-py = "^0.14.0"
 boto3 = "^1.24.31"
 python-magic = "^0.4.27"
-turbinia = "20220216"
+turbinia = {version = "20220216", optional = true}
```

Comment:
My (maybe flawed) understanding of how this works is that you define the package as optional in ```[tool.poetry.dependencies]``` so that all of its dependencies are also flagged as optional when you run ```poetry lock```. Then you add the extras section with a name that can be used when installing (e.g. poetry install -E turbinia_legacy will install the packages defined in the turbinia_legacy list under extras).